Classes/Hooks/DefaultUploadFolder.php

```diff
@@ -33,7 +33,8 @@ public function getDefaultUploadFolder($params, BackendUserAuthentication $backe
         $uploadFolder = $params['uploadFolder'];
         $table = $params['table'];
         $field = $params['field'];
-        $pageTs = BackendUtility::getPagesTSconfig($params['pid']);
+        $pid = $params['pid'] ?? abs(array_keys($_GET['edit'][$table])[0]);
```

Comment:
@mbrodala can we be sure the first array key holds the pid? What if we are not in edit context?

Reply:
TBH I just updated the patch from #7 to have it cleanly re-apply. I'll need to check for the exact issue this was meant to fix back then.